utkarsh component/button/button.css

```diff
@@ -0,0 +1,355 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Thai+Looped:wght@300&display=swap');
+
+
+
+
+    
+
+
+    body{
+      font-family: 'IBM Plex Sans Thai Looped', sans-serif;
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  background-color:  #31353a;
+  height: 100%;
+  
+  }
+  
+  * {
+     box-sizing: border-box;
+  --primary-color: #ecfeff ;
+  --secondary-color: #4908fa;
+  }
+  
+nav{
+  background: #0082e6;
+  height: 80px;
+  width: 100%; 
+}
+label.logo{
+  color: #fff;
+  font-size: 35px;
+  line-height: 80px;
+  padding: 0 100px;
+  font-weight: bold;
+}
+nav .main-list{
+  float: right;
+  margin-right: 20px;
+}
+nav .main-list li{
+  display: inline-block;
+  line-height: 30px;
+  /* margin: 50px 0; */
+  top: 2px;
+}
+nav .main-list li a{
+  color: #fff;
+  font-size: 17px;
+  text-transform: uppercase;
+}
+a.active,a:hover{
+  background: #1b9bff;
+  transition: .5s;
+}
+.checkbtn{
+  font-size: 30px;
+  color: #fff;
+  float: right;
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+#check{
+  display: none;
+}
+@media (max-width: 925px){
+  label.logo{
+    font-size: 30px;
+    padding-left: 50px;
+  }

```

Comment:
kindly remember! never put media queries in between it is not a professional way of using media queries